### PR TITLE
Add form to manually generate custom meeting-excerpts and store them in a dossier.

### PR DESCRIPTION
**Pull request by deiferni**
This PR contains two new features:
- A form to manually generate custom excerpts and store them in a dossier.
- Markdown support in sablon templates.

It also contains some minor fixes and refactorings.

### CODE REVIEW

**Comment by deiferni** [comment]
@phgross plz have a look

File: versions.cfg [hunk 1]
```diff
@@ -21,3 +21,7 @@ mocker = 1.1.1
 # which causes test isolation problems with PloneTestCase layers
 # which are not isolating properly.
 zope.testrunner = 4.4.4
+
+
+[ruby-versions]
+sablon = 0.0.14
```
Comment (phgross) [inline versions.cfg]: Not necessary already in KGS ...

**Comment by phgross** [comment]
Something with the updated `protokollauszug.docx` is wrong 
bildschirmfoto 2015-04-21 um 16 05 38

File: opengever/meeting/model/meeting.py [hunk 2]
```diff
@@ -101,6 +108,8 @@ class Meeting(Base):
         'GeneratedProtocol', uselist=False,
         backref=backref('meeting', uselist=False),
         primaryjoin="GeneratedProtocol.document_id==Meeting.protocol_document_id")
+    excerpt_documents = relationship('GeneratedExcerpt',
```
Comment (phgross) [inline opengever/meeting/model/meeting.py]: why ist that relation an N:N ?

File: opengever/meeting/model/agendaitem.py [hunk 3]
```diff
@@ -49,11 +100,36 @@ def remove(self):
...
+    @property
+    def considerations(self):
+        return self.proposal.considerations if self.has_proposal else None
+
+    @property
+    def proposed_action(self):
+        return self.proposal.proposed_action if self.has_proposal else None
+
+    @property
+    def name(self):
```
Comment (phgross) [inline opengever/meeting/model/agendaitem.py]: should be `agenda_item``

Could you add a small comment.

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po [hunk 4]
```diff
@@ -36,38 +36,42 @@ msgid "Add Member"
...
-#: ./opengever/meeting/browser/meetings/protocol.py:144
+#: ./opengever/meeting/browser/meetings/excerpt.py:141
+#: ./opengever/meeting/browser/meetings/protocol.py:145
 #: ./opengever/meeting/form.py:50
 msgid "Cancel"
 msgstr "Abbrechen"
 
+#: ./opengever/meeting/browser/meetings/templates/excerpt.pt:23
+msgid "Choose Agenda Items"
+msgstr "Traktanded auswählen"
```
Comment (phgross) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: Traktanden auswählen

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po [hunk 5]
```diff
@@ -129,6 +145,30 @@ msgstr "Protokoll generieren"
...
+msgid "Include considerations"
+msgstr "Überlegungen einfügen"
+
+#: ./opengever/meeting/browser/meetings/excerpt.py:58
+msgid "Include decision"
+msgstr "Entscheid einfügen"
+
+#: ./opengever/meeting/browser/meetings/excerpt.py:54
+msgid "Include discussion"
+msgstr ""
```
Comment (phgross) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: Missing translation

File: versions.cfg (hunk 1 above)
Comment (deiferni) [inline versions.cfg]: oh thanks, i forgot i updated the KGS in the meantime :wink:

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po (hunk 5 above)
Comment (deiferni) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: fixed.

File: opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po (hunk 4 above)
Comment (deiferni) [inline opengever/meeting/locales/de/LC_MESSAGES/opengever.meeting.po]: fixed.

File: opengever/meeting/model/agendaitem.py (hunk 3 above)
Comment (deiferni) [inline opengever/meeting/model/agendaitem.py]: :+1:

File: opengever/meeting/model/meeting.py (hunk 2 above)
Comment (deiferni) [inline opengever/meeting/model/meeting.py]: Correct, this is not a true many-to-many relationship. I'd like to keep `GeneratedDocument`'s as simple as possible though and avoid that it actively knows about its relationships (i.e. add a foreign-key column), currently it is used in many places already and usage will grow.

**Comment by deiferni** [comment]
Protokollauszug.docx has been fixed.

**Comment by deiferni** [comment]
@phgross updated

**Pull request merged**